src/app.js

```diff
@@ -69,4 +79,20 @@ app.post(
   require('./middleware/publishUpload')(pub)
 );
 
+// Routes: Debug - Show Environment and App Variables
+app.get('/debug/app-vars', (req, res) => res.json({
```

Comment:
Seems incredibly insecure to leave this in place. Comment out, maybe, or only provide if `process.env.NODE_ENV === 'development'`?

Reply:
The intended usage of this app is for students to download and run it themselves (passing in their own environments as explained via the course). So are the developers in this context. However, if we ever did host this ourselves then yes that's a bad thing. I'll do as suggested here ^

Reply:
If nothing else we need to set a good example for our students :)